docs/releases/4.2.txt

```diff
@@ -155,7 +155,8 @@ Email
 Error Reporting
 ~~~~~~~~~~~~~~~
 
-* ...
+* For Python 3.11+ the ``technical_500.html`` debug page now also highlights the columns in the 
+  line in which an error occurred.
```

Comment:
Re-reading this, I think we can use the same language as Python, and there’s no need to mention the template name which I imagine most users don’t know about.

Plus, we show wrap at 80 chars

```suggestion
* The debug page now shows Python 3.11’s fine-grained error locations.
```